### DISCUSSION TITLE
Enterprise updates?

### DISCUSSION THREAD

**Discussion by NiklasBr** [pimcore/pimcore#17833] [General]
> `Upgrading pimcore/pimcore (v11.4.1 => v11.4.1.2)`

Are there any release notes available to send to customers who ask what has changed between versions?

**Comment by fashxp** [comment]
For all future patch releases we will add release-notes in the corresponding enterprise repositories where we provide patch releases for enterprise customers, e.g. here https://github.com/pimcore/ee-pimcore/releases/tag/v11.4.1.2

Partners as well as enterprise customers can get a github user and access these repositories directly at github. 
Furthermore we are thinking about providing one central (public) place where we collect all release notes of all releases of all repositories. But we need to check, how this can be done...

**Reply by NiklasBr** [reply]
Thanks! I'll create a new account and request access.

**Reply by NiklasBr** [reply]
I created a github user to access them: @NiklasBr-ateles

**Comment by NiklasBr** [comment]
Thanks! 🌟
